README.md

```diff
@@ -53,11 +53,12 @@ $ npm install --save-dev backbone.react-bridge
       // Customize the form of the properties which will be passed to the
       // React Component. In case that 'getProps' is undefined, a composition
       // of the model's attributes, the collection's values and the custom
-      // properties will be returned to the React Component
+      // properties will be returned to the React Component.
+      // `getProps` receives an object with `model` and `collection` as members.
```

Comment:
Nit: ...receives an object with `model` and `collection` as **properties**

Reply:
Updated